Client/mods/deathmatch/logic/CCustomData.cpp

```diff
@@ -14,35 +14,40 @@
 
 void CCustomData::Copy(CCustomData* pCustomData)
 {
-    std::map<std::string, SCustomData>::const_iterator iter = pCustomData->IterBegin();
+    CFastHashMap<SString, SCustomData>::const_iterator iter = pCustomData->IterBegin();
     for (; iter != pCustomData->IterEnd(); iter++)
     {
-        Set(iter->first.c_str(), iter->second.Variable);
+        Set(iter->first, iter->second.Variable);
     }
 }
 
 SCustomData* CCustomData::Get(const char* szName)
 {
-    assert(szName);
-
-    std::map<std::string, SCustomData>::iterator it = m_Data.find(szName);
+    CFastHashMap<SString, SCustomData>::iterator it = m_Data.find(szName);
     if (it != m_Data.end())
         return &it->second;
 
     return NULL;
```

Comment:
Maybe just replace the content of `Get` with a call with `MapFind`?

Reply:
We could but that'd be just a pointless function call, and we can to squeeze out as much performance as we can, and I think this isn't premature optimization(yet)

Reply:
There's no real difference

Reply:
Isn't it converting from SString to const char* and then back?
Or it's optimized by the compiler already?

Reply:
99% chance optimised away. If in doubt, measure the difference